### DISCUSSION TITLE
Overriding router tabs styles

### DISCUSSION THREAD

**Discussion by lazaroofarrill** [quasarframework/quasar#9936] [General - Components / Directives / etc] [answered]
I wan't to make router tabs a lot more narrow and change the indicator shape and animation. Are there any clases to override this behaviors or do I need to implement my own tabs?

**Comment by ontwikkelfabriek** [comment, marked as answer]
@lazaroofarrill 

No you don't need your own tabs. All components are styled with css using BEM strategy.  Use the chrome/firefox debugger see what classes control the looks of the components.

BEM:
http://getbem.com/introduction/

here's an example of customizing a Quasar component. 
https://codepen.io/Hawkeye64/pen/WNNdKJV

**Reply by lazaroofarrill** [reply]
Thanks a lot, I really need to learn to make a more effective use of the inspector.
